src/menv/commands/create.py

```
@@ -54,6 +54,85 @@
+    },
+    # Future extensions can be added here
+    # {
+    #     "tag": "ollama-models",
+    #     "name": "LLM Models",
+    #     "description": "Download large LLM models for Ollama (Requires heavy network usage)",
+    #     "profile_specific": False,
+    # },
+]
+
+
+def _print_optional_tasks_summary(profile: str) -> None:
```

Ruff (0.14.10)

Unused function argument: `profile`

(ARG001)